examples/ec2/v1beta1/fleetrequest.yaml

```
@@ -0,0 +1,89 @@
+# SPDX-FileCopyrightText: 2024 The Crossplane Authors <https://crossplane.io>
+#
+# SPDX-License-Identifier: CC0-1.0
+
+apiVersion: ec2.aws.upbound.io/v1beta1
+kind: FleetRequest
+metadata:
+  annotations:
+    meta.upbound.io/example-id: ec2/v1beta1/fleetrequest
+  labels:
+    testing.upbound.io/example-name: xplane_se
+  name: xplane-se
+spec:
+  forProvider:
+    region: us-west-2
+    type: instant
+    launchTemplateSpecification:
+      launchTemplateId: xplane-se
+      version: 1
+      overrides:
+        - subnetId: s
+          instanceType: t3.micro
+    onDemandOptions:
+      minTargetCapacity: 1
+      singleAvailabilityZone: true
+      singleInstanceType: true
+    replaceUnhealthyInstances: true
+    targetCapacitySpecification:
+      defaultTargetCapacityType: on-demand
+      onDemandTargetCapacity: 1
+      totalTargetCapacity: 1
+    tagSpecifications:
+      - resourceType: instance
+        tags:
+          dremio_managed: true
+      - resourceType: volume
+        tags:
+          dremio_managed: true
+      - resourceType: fleet
+        tags:
+          dremio_managed: true
```

```suggestion
    launchTemplateConfig:
    - launchTemplateSpecification:
        launchTemplateIdSelector:
          matchLabels:
            testing.upbound.io/example-name: example
        versionSelector:
          matchLabels:
            testing.upbound.io/example-name: example
    region: us-west-1
    targetCapacitySpecification:
      defaultTargetCapacityType: spot
      totalTargetCapacity: 5
```

We can try the simple example at the beginning to test it with uptest. What do you think?